ci/conan/all/conanfile.py

```diff
@@ -266,10 +266,7 @@ def _with_boost(self, required=False):
             if self.options.gandiva:
                 return True
             version = tools.Version(self.version)
-            if version.major == "1":
-                if self._parquet() and self.settings.compiler == "gcc" and self.settings.compiler.version < tools.Version("4.9"):
-                    return True
-            elif version.major >= "2":
+            if version.major >= "2":
```

Comment:
Could you revert all changes in this file?
A Conan package supports multiple versions in the same file. So we still need to support old versions in this file.